docs/assignment2.md

```diff
@@ -1 +1,409 @@
 # Assignment 2
+
+Number of words in this document: 8000
+
+**Format**: establish formatting conventions when  describing your models in this document. For example, you style the name of each class is *Italics*, whereas the attributes, operations, and  associations as <u>underlined</u> text, objects are in italic, etc.
+
+###
+
+### Implemented feature
+
+| ID   | Short name | Description                                                  |
```

Comment:
As suggested in the feedback above, it is necessary clarifications about the gameplay structure, game objectives and so forth.